packages/server/graphql/types/TeamPromptMeeting.ts

```diff
@@ -41,6 +43,49 @@ const TeamPromptMeeting = new GraphQLObjectType<any, GQLContext>({
         const meetingMember = await dataLoader.get('meetingMembers').load(meetingMemberId)
         return meetingMember || null
       }
+    },
+    taskCount: {
+      type: new GraphQLNonNull(GraphQLInt),
+      description: 'The number of tasks generated in the meeting',
+      resolve: async ({id: meetingId}, _args: unknown, {dataLoader}: GQLContext) => {
+        const meeting = await dataLoader.get('newMeetings').load(meetingId)
+        if (meeting.meetingType !== 'teamPrompt') {
+          return 0
+        }
+        const {phases} = meeting
+        const discussPhase = getPhase(phases, 'RESPONSES')
+        const {stages} = discussPhase
+        const discussionIds = stages.map((stage) => stage.discussionId)
+        const r = await getRethink()
+        return r
+          .table('Task')
+          .getAll(r.args(discussionIds), {index: 'discussionId'})
+          .count()
+          .default(0)
+          .run()
+      }
+    },
+    commentCount: {
+      type: new GraphQLNonNull(GraphQLInt),
+      description: 'The number of comments generated in the meeting',
+      resolve: async ({id: meetingId}, _args: unknown, {dataLoader}: GQLContext) => {
+        const meeting = await dataLoader.get('newMeetings').load(meetingId)
+        if (meeting.meetingType !== 'teamPrompt') {
+          return 0
+        }
+        const {phases} = meeting
+        const discussPhase = getPhase(phases, 'RESPONSES')
+        const {stages} = discussPhase
+        const discussionIds = stages.map((stage) => stage.discussionId)
+        const r = await getRethink()
+        return r
```

Comment:
-1 these aggregation queries actually end up being _real_ expensive since we fetch them for the timeline event. IIRC we shaved off almost a whole second on the load time for the timeline view by caching these.
Can we cache them on the meeting object like we do for the other meeting types?

Reply:
@mattkrick These actually only run for the summary itself - there are similar resolvers for these on the existing timeline event.

We _could_ cache these, but the result would be inaccurate if any of the stats change after the meeting is ended (which is quite possible for things like standup discussions), and we made the decision that the timeline should reflect the current state, not the state when the meeting was ended.  See https://github.com/ParabolInc/parabol/issues/5927#issuecomment-1128113425

Reply:
> These actually only run for the summary itself
Any access to a meeting object will be able to access these, and the timeline event links to the meeting object

> there are similar resolvers for these on the existing timeline event.

could you include a link to where you see this? This field is cached for the other 3 meeting types, e.g.:
https://github.com/ParabolInc/parabol/blob/8409e5ee0ab86c9effcb616dd82cdf30053c81ee/packages/server/graphql/types/ActionMeeting.ts#L52

> we made the decision that the timeline should reflect the current state

we'll need to either undo that decision or create dataloaders for these queries & update the other 3 meeting types so they all have similar behavior. With the number of timeline events that a standup will be creating, I'd opt for the former since each timeline event is going to have to run these aggregation queries & we're gonna run into performance issues again.

Reply:
That's fair.

In any case, this issue is somewhat tangential to this PR, since the timeline view currently uses [different resolvers](https://github.com/ParabolInc/parabol/blob/50f7d8268eff5906f6b6a849f6fef1060f49d28d/packages/server/graphql/public/types/TimelineEventTeamPromptComplete.ts#L25-L65), so I'd prefer to address this in a separate issue.

Reply:
ohhhhh shoot we em in 2 places!
so we've got `taskCount, responseCount, commentCount` that should all live in 1 place together.
I'd say that place should be on the TeamPromptMeeting object, since those are aggregation stats about the meeting & in the future we might want to access those without accessing the timelineevent.

What we don't want is having them exist on 2 objects (TimelineEvent & Meeting) because that removes the single source of truth.

I'm all for moving the caching logic to a new PR since that'll be a little more involved. For this PR could you remove those fields from the timelineEvent object so this PR doesn't introduce a 2nd source of truth?

Reply:
Ahh that makes a lot of sense.  Done.